src/components/VInput/VInput.vue

```diff
@@ -66,6 +90,18 @@
         {{ label }}
       </span>
 
+      <slot
+        name="label"
+        v-bind="{
+          valid,
+          dirty,
+          error,
+          invalid,
+          anyInvalid,
+          errors: errorMessages,
+        }"
+      />
+
```

Comment:
```suggestion
      <slot
        name="label"
        v-bind="{
          valid,
          dirty,
          error,
          invalid,
          anyInvalid,
          errors: errorMessages,
        }"
      >
        <span :class="['vts-input__text', classes.text]">
          {{ label }}
        </span>
      </slot>

```

(GitHub didnt let me highlight the opening span tag in the suggestion, so this is technically broken)